crates/ruff_linter/src/codes.rs

```diff
@@ -285,6 +285,7 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         #[allow(deprecated)]
         (Pylint, "W3201") => (RuleGroup::Nursery, rules::pylint::rules::BadDunderMethodName),
         (Pylint, "W3301") => (RuleGroup::Stable, rules::pylint::rules::NestedMinMax),
+        (Pylint, "W9011") => (RuleGroup::Stable, rules::pylint::rules::MissingReturnDoc),
```

Comment:
We add new rules in preview first

```suggestion
        (Pylint, "W9011") => (RuleGroup::Preview, rules::pylint::rules::MissingReturnDoc),
```

Reply:
That makes a lot of sense, I missed it when reading `CONTRIBUTING.md`. Thanks for catching!